src/test/java/seedu/guest/logic/commands/ResetRoomCleanCommandTest.java

```diff
@@ -0,0 +1,75 @@
+package seedu.guest.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.guest.logic.commands.CommandTestUtil.VALID_IS_ROOM_CLEAN_BOB;
+import static seedu.guest.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.guest.testutil.TypicalGuests.getTypicalGuestBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.guest.model.Model;
+import seedu.guest.model.ModelManager;
+import seedu.guest.model.UserPrefs;
+import seedu.guest.model.guest.Guest;
+import seedu.guest.testutil.GuestBuilder;
+
+
+
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for ResetRoomCleanCommand.
+ */
+public class ResetRoomCleanCommandTest {
+
+    private Model model = new ModelManager(getTypicalGuestBook(), new UserPrefs());
+
+    @Test
+    public void execute_resetRoomCleanCommand_success() {
+        Guest guestInFilteredList0 = model.getFilteredGuestList().get(0);
+        Guest guestInFilteredList1 = model.getFilteredGuestList().get(1);
+        Guest guestInFilteredList2 = model.getFilteredGuestList().get(2);
+        Guest guestInFilteredList3 = model.getFilteredGuestList().get(3);
+        Guest guestInFilteredList4 = model.getFilteredGuestList().get(4);
+        Guest guestInFilteredList5 = model.getFilteredGuestList().get(5);
+        Guest guestInFilteredList6 = model.getFilteredGuestList().get(6);
+
+        Guest editedGuest0 = new GuestBuilder(guestInFilteredList0).withIsRoomClean(VALID_IS_ROOM_CLEAN_BOB).build();
+        Guest editedGuest1 = new GuestBuilder(guestInFilteredList1).withIsRoomClean(VALID_IS_ROOM_CLEAN_BOB).build();
+        Guest editedGuest2 = new GuestBuilder(guestInFilteredList2).withIsRoomClean(VALID_IS_ROOM_CLEAN_BOB).build();
+        Guest editedGuest3 = new GuestBuilder(guestInFilteredList3).withIsRoomClean(VALID_IS_ROOM_CLEAN_BOB).build();
+        Guest editedGuest4 = new GuestBuilder(guestInFilteredList4).withIsRoomClean(VALID_IS_ROOM_CLEAN_BOB).build();
+        Guest editedGuest5 = new GuestBuilder(guestInFilteredList5).withIsRoomClean(VALID_IS_ROOM_CLEAN_BOB).build();
+        Guest editedGuest6 = new GuestBuilder(guestInFilteredList6).withIsRoomClean(VALID_IS_ROOM_CLEAN_BOB).build();
```

Comment:
Should we have some cleaned and uncleaned rooms to test that the command works for both?

Reply:
Yes, you are right, thanks!